config.template/faf-exe-upload/faf-exe-upload.env

```diff
@@ -0,0 +1,3 @@
+API_URL=http://api.test.faforever.com
```

Comment:
The default should always point to the localhost service, in this case `http://faf-java-api:8010`